cluster/addons/cluster-loadbalancing/glbc/README.md

```diff
@@ -1,112 +1,9 @@
 # GCE Load-Balancer Controller (GLBC) Cluster Addon
 
-This cluster addon is composed of:
-* A [Google L7 LoadBalancer Controller](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/gce)
-* A [404 default backend](https://github.com/kubernetes/contrib/tree/master/404-server) Service + RC
+This cluster addon is composed of a 404 default backend service and deployment.
+On GCE, this 404 service is used in conjuction with the
```

Comment:
Turned out a typo failed the test :)
conjuction -> conjunction

Reply:
Done